keyboards/yandrstudio/transition80/transition80.c

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Y&R-Biu (@jiaxin96)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "transition80.h"
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    if (!process_record_user(keycode, record)) { return false; }
+    switch(keycode) {
+        case KC_LG:
+            if (record->event.pressed) {
+                process_magic(GUI_TOG, record);
+            }
+            return false;
+        case KC_MACOS:
+            if (record->event.pressed) {
+                process_magic(CG_TOGG, record);
+            }
+            return false;
```

Comment:
Why is this reimplementing core functionality?

Reply:
In order to display these keys in VIA, and the definitions of these keys have been put into via/keymap.c now.

Reply:
These keycode have static values, and this code is unnecessary and should be removed. If this needs to be fixed in via, then thats not a problem for this repo, and should be fixed there instead.

Reply:
Well, it is really difficult to wait or push the update of via. Anyway, I have deleted this code according to your suggestion.